main.go

```diff
@@ -160,6 +175,19 @@ func main() {
 	coordinater.Run()
 }
 
+func parseAddress(address string) (string, error) {
```

Comment:
Actually we should use https://golang.org/pkg/net/#SplitHostPort since SplitAddress allows for a the `proto:` prefix that we're not using in its input. But neither accepts a plain port number, without the preceding `:`.